### PR TITLE
Move poetryOverrides to lib

### PR DESCRIPTION
**Pull request by hcallahan-lowrisc**
API change, so requires a change in downstream flakes consuming this. [[1]](https://github.com/lowRISC/ibex-demo-system/pull/83)
This removes the following warning from nix flake check:

```
warning: unknown flake output 'poetryOverrides'
```

### CODE REVIEW

File: lib/poetryOverrides.nix [hunk 1]
```diff
@@ -43,10 +43,19 @@
...
+    #      # This fix is incomplete, and appears to be due to the issue here:
+    #      # https://github.com/nix-community/poetry2nix/issues/413
+    #      # https://github.com/nix-community/poetry2nix/issues/442
+    #      buildInputs = (old.buildInputs or []) ++ [
+    #        pkgs.rustc
+    #        pkgs.cargo
+    #        # pkgs.rustPlatform.cargoSetupHook
+    #        final.setuptools-rust
+    #      ];
+    #    });
```
Comment (nbdd0121) [inline lib/poetryOverrides.nix:58]: What package is this comment for?

Comment (hcallahan-lowrisc) [reply lib/poetryOverrides.nix:58]: libcst, the one above

Comment (nbdd0121) [reply lib/poetryOverrides.nix:58]: Can you add a comment or maybe move the libcst semicolon after the comment?

Comment (hcallahan-lowrisc) [reply lib/poetryOverrides.nix:57]: Done

**Review by nbdd0121 — APPROVED** [review]

**Comment by nbdd0121** [comment]
Ah nix fmt isn't happy

**Pull request merged**